tests/functional/test_data_classes.py

```diff
@@ -617,6 +631,23 @@ def test_dynamo_attribute_value_type_error():
         print(attribute_value.get_type)
 
 
+def test_stream_record_keys_with_valid_keys():
+    attribute_value = {"Foo": "Bar"}
+    sr = StreamRecord({"Keys": {"Key1": attribute_value}})
+    assert sr.keys == {"Key1": AttributeValue(attribute_value)}
+
+
+def test_stream_record_keys_with_no_keys():
+    sr = StreamRecord({})
+    assert sr.keys is None
+
+
+def test_stream_record_keys_overrides_dict_wrapper_keys():
+    data = {"Keys": {"key1": {"attr1": "value1"}}}
+    sr = StreamRecord(data)
+    assert sr.keys != data.keys()
```

Comment:
```suggestion
def test_stream_record_keys_with_valid_keys():
    attribute_value = {"Foo": "Bar"}
    record = StreamRecord({"Keys": {"Key1": attribute_value}})
    assert record.keys == {"Key1": AttributeValue(attribute_value)}


def test_stream_record_keys_with_no_keys():
    record = StreamRecord({})
    assert record.keys is None


def test_stream_record_keys_overrides_dict_wrapper_keys():
    data = {"Keys": {"key1": {"attr1": "value1"}}}
    record = StreamRecord(data)
    assert record.keys != data.keys()
```

np: better describes intent. Over time, we tend to forget those short letter variable names. Any contributor with OCD or ADHD might unintentionally obsess over its meaning too.